lib/utils.sh

```diff
@@ -29,21 +29,24 @@ ensure_ruby_build_installed() {
 download_ruby_build() {
     # Print to stderr so asdf doesn't assume this string is a list of versions
     echo "Downloading ruby-build..." >&2
-    local build_dir="ruby-build-source"
+    local build_dir="$(asdf_ruby_plugin_path)/ruby-build-source"
 
     # Clone down and checkout the correct ruby-build version
     git clone https://github.com/rbenv/ruby-build.git $build_dir >&2 >/dev/null
     (cd $build_dir; git checkout $RUBY_BUILD_TAG >&2 >/dev/null)
```

Comment:
The problem seems to be with this `cd` and my `CDPATH`.
My suggestion is to use the full path for the `ruby-build-source`.